app/src/main/java/com/quran/labs/androidquran/ui/fragment/TabletFragment.java

```diff
@@ -320,15 +321,17 @@ protected void onPostExecute(RectF[] rect) {
             mRightImageView.setPageBounds(rect[0]);
             mLeftImageView.setPageBounds(rect[1]);
             Context context = getContext();
-            if (mOverlayText && context != null) {
+            if (mOverlayText && isAdded()) {
```

Comment:
+1! Now, since context is not used outside, you could move `Context context = getContext();` to inside the brackets (your original change). Sorry for sounding like nitpicking :(